### DISCUSSION TITLE
Lack of power supply via RS485 plug

### DISCUSSION THREAD

**Discussion by arpiska** [Gifford47/HCPBridgeMqtt#96] [Q&A]
Hello,

Before I connected my own build, I measured the voltage between pins 1-2 and 5-6 and there is no 24V
The cable is ok, I measured it with a multimeter.

What could be the problem?
Do I need to turn on somewhere?

**Comment by arpiska** [comment]
SupraMatic P4

**Comment by Gifford47** [comment]
The power is only supplied by supramatic after a successful bus scan.

**Reply by arpiska** [reply]
Thank you Gifford!  During BUS scan, do I need to apply separate power supply from USB or i just have to wait?

**Reply by Gifford47** [reply]
You only have to wait until bus scan is complete. Afterwards the unit is supplied by supramatic continuously. You find this instruction also in the readme

**Reply by arpiska** [reply]
Connected the board to Supramatic, it worked for a short time. Then does not respond to commands and blocks the engine and can't be opened even with the wall button.
After de-energizing for 15min and deconnect the board, it is working normally.
I read that the esphome version works correctly, please could you help me how to flash it?
